### DISCUSSION TITLE
Do you have a compiler for .uf2?

### DISCUSSION THREAD

**Discussion by Mr-Amnat** [markadev/AppleII-VGA#20] [Q&A] [answered]
Please help share .uf2 pi pico vga.
I can't compile it.
Thank you.

**Comment by markadev** [comment, marked as answer]
Yes! You can download the compiled .uf2 files from: https://github.com/markadev/AppleII-VGA/releases

Just choose the one for your Apple II model (II+ or IIe)

**Reply by Mr-Amnat** [reply]
**Thank you.**

**Reply by jakubpolomsky** [reply]
Do you happen to have builds for apple II europlus? I am trying to get it running and I tried both iie and iiplus files, yet all I can see are the inverted @ signs and a blinking cursor in four lines (cycling through the four cursor positions when I advance on the lines on the composite connected monitor)

**Reply by jakubpolomsky** [reply]
Oh nevermind... I realized I was using the 74LS instead of LV... now everything works perfectly fine

**Reply by markadev** [reply]
Cool, glad to hear you got it working!

**Reply by Ret-Marut** [reply]
Hi, I have the same problem (inverted @ only, and after some hours I see flickering some LowRes artifacts). I have LVC buffers installed, though, could the be fake ones? I ordered them from china...
Regards
Bernd

**Reply by markadev** [reply]
> Hi, I have the same problem (inverted @ only, and after some hours I see flickering some LowRes artifacts). I have LVC buffers installed, though, could the be fake ones? I ordered them from china... Regards Bernd

Hey there.  It's always possible they could be fake but you can also double-check the 3.3v supply on the ICs and if you have a scope then you can check pin 31 on the Pico to see that the Apple 1MHz bus clock signal is making it through.

**Comment by Ret-Marut** [comment]
Hi Mark,
sorry, it took a while (because of other projects ;-), and here are my findings:
- The 1MHz clock on the Pico side looks weird. See the scope picture (bottom is the Apple 1 MHz clock, 2V/div, top is at pin 4 of LVC245, 1V/div, both taken with 10:1 prescaler, triggering on bottom signal, neg). There seems to be a jitter on the 3V side.
- lifting pin 4 (i.e. disconnecting the Pico I/O) showed a marginal change of the signal
- blocking with additional caps on the supply didn't help
- testing the 74LVC245's on a breadboard revealed nothing strange
- verifying the board wiring and connections also didn't show anything strange (I'm using the v1.6)
I now ordered a set of new 74LVC245 from DigiKey, that's my last try then...

Regards
Bernd
![IMG_1971](https://github.com/user-attachments/assets/8c3a512b-993e-42d4-94a2-9f10bb1bd6a5)
![IMG_1970](https://github.com/user-attachments/assets/f629cf58-5822-4fc5-bf43-3892e0955bf0)
